SwaprHQ/swapr-algebra-ui
# [LIQ-34][CHORE] - Add SAFE icon

## Fixes: [LIQ-34](https://linear.app/swaprhq/issue/LIQ-34/please-add-safe-icon)

# Description

* Add missing pool token logos for Safe, Curve and Ajna


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to "Pools" header item and select any token to be approved for swap
4) You should see token icons for $AJNA, $CRV and $SAFE tokens
